index.html

```diff
@@ -16680,25 +16680,27 @@ <h3>Presentational Roles Conflict Resolution</h3>
             explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined.
           </li>
           <li>
-            If an element has <a>global</a> WAI-ARIA states or properties, user agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and instead expose the element's implicit role.
-            However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit
-            non-presentational role is applied.
+              <p>If an element has <a>global</a> WAI-ARIA states or properties, the behavior depends on whether the presentational role is explicit or inherited:</p>
+              <ul>
+                <li>
+                  <strong>Explicit presentational role:</strong> User agents MUST ignore the <rref>none</rref>/<rref>presentation</rref> role and instead expose the element's implicit role.
+                </li>
+                <li>
+                  <strong>Inherited presentational role:</strong> User agents MUST NOT expose the element's role, maintaining its presentational role regardless of any <a>global</a> ARIA state or property.
+                </li>
+              </ul>
+              <p>However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element's role MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</p>
+
+            <p class="note">Some <a>global</a> WAI-ARIA states and properties are <a>prohibited</a> on certain roles. These states and properties are still considered global for the purposes of Presentational Role Conflict resolution.</p>
+            <p>For example, <pref>aria-describedby</pref> is a global attribute and would always be applied; <pref>aria-level</pref> is not a global attribute and would therefore only apply if the element was not in a presentational state.</p>
```

Comment:
Small edit to keep tenses consistent:

```suggestion
            <p>For example, <pref>aria-describedby</pref> is a global attribute and would always be applied; <pref>aria-level</pref> is not a global attribute and would therefore only apply if the element is not in a presentational state.</p>
```